### PR TITLE
[Backport 1.7.x] manual backport of #5990

### PR DESCRIPTION
**Pull request by christophd**
Fixed merge conflicts in

* app/ui-react/syndesis/src/modules/extensions/pages/ExtensionDetailsPage.tsx
* app/ui-react/syndesis/src/modules/extensions/pages/ExtensionsPage.tsx
* app/ui-react/syndesis/src/modules/extensions/shared/ExtensionIntegrations.tsx
* app/ui-react/syndesis/src/modules/integrations/components/TagIntegrationDialogWrapper.tsx
* app/ui-react/syndesis/src/modules/integrations/pages/cicd/ManageCiCdPage.tsx
* app/ui-react/syndesis/src/setupProxy.js

### CODE REVIEW

File: app/ui-react/syndesis/src/modules/extensions/pages/ExtensionDetailsPage.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ export default class ExtensionDetailsPage extends React.Component {
                         error={error}
                         loading={!hasData}
                         loaderChildren={<PageLoader />}
-                        errorChildren={<ApiError />}
+                        errorChildren={<ApiError error={error as Error} />}
```
Comment (riccardo-forina) [inline app/ui-react/syndesis/src/modules/extensions/pages/ExtensionDetailsPage.tsx]: Actually no, when used with a `WithXyz` component error should be the `errorMessage` property that comes from it. Like [here](https://github.com/syndesisio/syndesis/pull/6008/files#diff-7d7b882bd5054157b0365bad6d545241R156)

File: app/ui-react/syndesis/src/modules/extensions/pages/ExtensionsPage.tsx [hunk 2]
```diff
@@ -149,7 +149,7 @@ export default class ExtensionsPage extends React.Component {
                                     }}
                                   />
                                 }
-                                errorChildren={<ApiError />}
+                                errorChildren={<ApiError error={error as Error} />}
```
Comment (riccardo-forina) [inline app/ui-react/syndesis/src/modules/extensions/pages/ExtensionsPage.tsx]: Same here.

File: app/ui-react/syndesis/src/modules/extensions/shared/ExtensionIntegrations.tsx [hunk 3]
```diff
@@ -40,7 +40,7 @@ export default class ExtensionIntegrations extends React.Component<
             error={error}
             loading={!hasData}
             loaderChildren={<Loader />}
-            errorChildren={<ApiError />}
+            errorChildren={<ApiError error={error as Error} />}
```
Comment (riccardo-forina) [inline app/ui-react/syndesis/src/modules/extensions/shared/ExtensionIntegrations.tsx]: and here

File: app/ui-react/syndesis/src/modules/extensions/pages/ExtensionDetailsPage.tsx (hunk 1 above)
Comment (christophd) [reply app/ui-react/syndesis/src/modules/extensions/pages/ExtensionDetailsPage.tsx]: Thanks! I have added a new commit hopefully fixing things

File: app/ui-react/syndesis/src/modules/extensions/pages/ExtensionsPage.tsx (hunk 2 above)
Comment (christophd) [reply app/ui-react/syndesis/src/modules/extensions/pages/ExtensionsPage.tsx]: fixed with additional commit

File: app/ui-react/syndesis/src/modules/extensions/shared/ExtensionIntegrations.tsx (hunk 3 above)
Comment (christophd) [reply app/ui-react/syndesis/src/modules/extensions/shared/ExtensionIntegrations.tsx]: fixed with additional commit

**Pull request merged**